src/app/pages/interest-form/interest-form.hbs

```diff
@@ -0,0 +1,203 @@
+    <fieldset class="labeled-inputs">
+        <label>
+            <span>First name</span>
+            <input name="adoption_interest[first_name]" type="text" required="true" />
+        </label>
+        <label>
+            <span>Last name</span>
+            <input name="adoption_interest[last_name]" type="text" required="true" />
+        </label>
+        <label>
+            <span>School name</span>
+            <input name="adoption_interest[school_name]" type="text" required="true" />
+        </label>
+        <label>
+            <span>School url</span>
+            <input name="adoption_interest[school_url]" type="text" required="true" />
+        </label>
+        <label>
+            <span>Email address</span>
+            <input name="adoption_interest[email_address]" type="email"
+             pattern="[^@]+@([^@\.]+\.)+[^@\.]+" required="true" />
+        </label>
+        <label>
+            <span>Phone number</span>
+            <input name="adoption_interest[phone_number]" type="text" required="true" />
+        </label>
+    </fieldset>
+    <h2>Textbook Interest</h2>
+    <p>OpenStax College textbook are you interested in adopting:</p>
+    <fieldset class="labeled-buttons require-one">
+        <legend class="validation-message">
+            Select at least one
+        </legend>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Algebra and Trigonometry">
+            Algebra and Trigonometry
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Anatomy and Physiology">
+            Anatomy and Physiology
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Biology">
+            Biology
+        </label>
+        <label>
+            <input id="adoption_interest_textbook_interests_" name="adoption_interest[textbook_interests][]" type="checkbox" value="Chemistry">
+            Chemistry
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="College Algebra">
+            College Algebra
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="College Physics">
+            College Physics
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="College Physics for AP® Courses ">
+            College Physics for AP® Courses
+        </label>
+        <label>
+            <input name="adoption_interest[textbook_interests][]" type="checkbox" value="Concepts of Biology">
+            Concepts of Biology
+        </label>
+        <label>
+            <input id="adoption_interest_textbook_interests_" name="adoption_interest[textbook_interests][]" type="checkbox" value="Introduction to Sociology">
```

Comment:
A number of elements have identical ids, which is not valid HTML.  For example, I'm seeing 3 different elements with `adoption_interest_textbook_interests_`.  It doesn't appear this ids are being used for anything either, in which case they should just be removed.